modules/accounts/src/lib.rs

```diff
@@ -280,6 +280,7 @@ impl<T: Trait> Module<T> {
 			// by treasury account is sufficient to open an account.
 			if *k != treasury_account {
 				// send dust native currency to treasury account
+				// it currently cannot fail, so ignore the result
```

Comment:
we transfer all free balances from a new account, so it shouldn't fail.
but even it failed, leave some dust storage is not a critical issue